core/src/dird/ua_update.cc

```diff
@@ -613,41 +615,41 @@ static bool UpdateVolume(UaContext* ua)
     MediaDbRecord mr;
     StorageDbRecord sr;
 
-    StartPrompt(ua, _("Parameters to modify:\n"));
-    AddPrompt(ua, _("Volume Status"));              /* 0 */
-    AddPrompt(ua, _("Volume Retention Period"));    /* 1 */
-    AddPrompt(ua, _("Volume Use Duration"));        /* 2 */
-    AddPrompt(ua, _("Maximum Volume Jobs"));        /* 3 */
-    AddPrompt(ua, _("Maximum Volume Files"));       /* 4 */
-    AddPrompt(ua, _("Maximum Volume Bytes"));       /* 5 */
-    AddPrompt(ua, _("Recycle Flag"));               /* 6 */
-    AddPrompt(ua, _("Slot"));                       /* 7 */
-    AddPrompt(ua, _("InChanger Flag"));             /* 8 */
-    AddPrompt(ua, _("Volume Files"));               /* 9 */
-    AddPrompt(ua, _("Pool"));                       /* 10 */
-    AddPrompt(ua, _("Volume from Pool"));           /* 11 */
-    AddPrompt(ua, _("All Volumes from Pool"));      /* 12 */
-    AddPrompt(ua, _("All Volumes from all Pools")); /* 13 */
-    AddPrompt(ua, _("Enabled")),                    /* 14 */
-        AddPrompt(ua, _("RecyclePool")),            /* 15 */
-        AddPrompt(ua, _("Action On Purge")),        /* 16 */
-        AddPrompt(ua, _("Storage")),                /* 17 */
-        AddPrompt(ua, _("Done"));                   /* 18 */
-    i = DoPrompt(ua, "", _("Select parameter to modify"), NULL, 0);
+    StartPrompt(ua, T_("Parameters to modify:\n"));
+    AddPrompt(ua, T_("Volume Status"));              /* 0 */
+    AddPrompt(ua, T_("Volume Retention Period"));    /* 1 */
+    AddPrompt(ua, T_("Volume Use Duration"));        /* 2 */
+    AddPrompt(ua, T_("Maximum Volume Jobs"));        /* 3 */
+    AddPrompt(ua, T_("Maximum Volume Files"));       /* 4 */
+    AddPrompt(ua, T_("Maximum Volume Bytes"));       /* 5 */
+    AddPrompt(ua, T_("Recycle Flag"));               /* 6 */
+    AddPrompt(ua, T_("Slot"));                       /* 7 */
+    AddPrompt(ua, T_("InChanger Flag"));             /* 8 */
+    AddPrompt(ua, T_("Volume Files"));               /* 9 */
+    AddPrompt(ua, T_("Pool"));                       /* 10 */
+    AddPrompt(ua, T_("Volume from Pool"));           /* 11 */
+    AddPrompt(ua, T_("All Volumes from Pool"));      /* 12 */
+    AddPrompt(ua, T_("All Volumes from all Pools")); /* 13 */
+    AddPrompt(ua, T_("Enabled")),                    /* 14 */
+        AddPrompt(ua, T_("RecyclePool")),            /* 15 */
+        AddPrompt(ua, T_("Action On Purge")),        /* 16 */
+        AddPrompt(ua, T_("Storage")),                /* 17 */
+        AddPrompt(ua, T_("Done"));                   /* 18 */
```

Comment:
```suggestion
    AddPrompt(ua, T_("Enabled"));                    /* 14 */
    AddPrompt(ua, T_("RecyclePool"));                /* 15 */
    AddPrompt(ua, T_("Action On Purge"));            /* 16 */
    AddPrompt(ua, T_("Storage"));                    /* 17 */
    AddPrompt(ua, T_("Done"));                       /* 18 */
```
Klappt zwar auch so wie es war, sieht aber komisch aus.